contrib/firmware_packager/install_dell_bios_exe.py

```diff
@@ -40,9 +40,19 @@ def parse_args():
     """Parse arguments for this client"""
     import argparse
 
-    parser = argparse.ArgumentParser(description="Interact with fwupd daemon")
+    parser = argparse.ArgumentParser(
+        description="Interact with fwupd daemon, temporarily disable lvfs signing to proceed installation"
+    )
     parser.add_argument("exe", nargs="?", help="exe file")
     parser.add_argument("deviceid", nargs="?", help="DeviceID to operate on(optional)")
+    parser.add_argument(
+        "-o",
+        "--off-signing",
+        dest="off_signing",
+        default=False,
+        action="store_true",
+        help="disable lvfs signing after the installation, enable if unspecified",
+    )
```

Comment:
I don't like off-signing.  I don't know what it means, even after reading the help string.  Maybe:

* `disable-fwupd-signing-requirement`
* default: `True`
* help: `disable daemon enforcement of signed CAB files`

Reply:
Hmm.. disabling the signing requirement is likely enforced otherwise can't move forward.  My original plan of this `arg` is to determine what value should be set after an installation.

Reply:
If you can read old value and just restore it you can drop this all together then

Reply:
I think you can rebase this off main again, right?